tests/UI/pages/BO/shopParameters/orderSettings/statuses/index.js

```diff
@@ -144,39 +166,42 @@ class Statuses extends BOBasePage {
    * @param page
    * @param row
    * @param columnName
-   * @param idColumn
+   * @param column
+   * @param tableName
    * @return {Promise<string>}
    */
-  async getTextColumn(page, row, columnName, idColumn) {
+  async getTextColumn(page, row, columnName, column, tableName = 'order') {
```

Comment:
I think tableName should be the second parameter like in other pages and without a default value (tests that already exist should be fixed too)

```suggestion
  async getTextColumn(page, tableName, row, columnName, column) {
```